src/server.ts

```diff
@@ -16,6 +16,16 @@ router.get("/nodes/:id", (ctx) => {
   ctx.body = getNodeAndChildren(Number(ctx.params.id))
 })
 
+router.post("/nodes", (ctx) => {
+  const body = ctx.request.query
+  ctx.body = storeNewNode(body.name, body.type, Number(body.parent))
+})
+
+router.del("/nodes/:ids", (ctx) => {
+  const ids = ctx.params.ids.split("-").map(Number)
```

Comment:
can't the params be sent in the body of the request somehow instead?

Reply:
I did some research & couldn't make it work using `ctx.request.body`

Reply:
https://stackoverflow.com/questions/22148087/request-body-is-undefined-in-koa

Reply:
I know, I did "yarn add koa-bodyparser" and couldn't make it work